src/scenes/home/informationForm/informationForm.js

```diff
@@ -16,6 +16,7 @@ class SignupInformation extends Component {
     super(props);
     this.onIdentifierStatusChange = this.onIdentifierStatusChange.bind(this);
     this.onCheckBoxChange = this.onCheckBoxChange.bind(this);
+    this.previousPage = this.previousPage.bind(this);
```

Comment:
Ah, I see. The bind in that function is necessary because it needs to be passed to a child component. As previousPage only mutates the state of the component it sits in, I don't believe it is necessary to bind it.